### PR TITLE
Quartair stratigrafie

### PR DESCRIPTION
**Pull request by pjhaest**
* [x] I have read and followed the guidelines in the `CONTRIBUTING` document
* [x] I have checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change
* [x] I have updated the documentation accordingly.
* [x] I have added tests to cover my changes.
* [x] All new and existing tests passed on python 2.7

Describe your fixes/additions/changes
Type of QuartaireStratigrafie added

### CODE REVIEW

File: pydov/search/interpretaties.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 from pydov.types.interpretaties import LithologischeBeschrijvingen
 from pydov.types.interpretaties import GecodeerdeLithologie
 from pydov.types.interpretaties import GeotechnischeCodering
+from pydov.types.interpretaties import QuartaireStratigrafie
```
Comment (Roel) [inline pydov/search/interpretaties.py]: Actually 'Quartairstratigrafie' is the better term. I would suggest changing it everywhere (except the WFS layer name) from 'quartaire stratigrafie' to 'quartairstratigrafie'.

```suggestion
from pydov.types.interpretaties import QuartairStratigrafie
```

File: pydov/search/interpretaties.py [hunk 2]
```diff
@@ -745,3 +746,126 @@ def search(self, location=None, query=None, return_fields=None):
...
+
+    __wfs_schema = None
+    __wfs_namespace = None
+    __md_metadata = None
+    __fc_featurecatalogue = None
+
+    def __init__(self):
+        """Initialisation."""
+        super(QuartaireStratigrafieSearch, self).__init__(
+            'interpretaties:quartaire_stratigrafie', QuartaireStratigrafie)
```
Comment (Roel) [inline pydov/search/interpretaties.py]: The WFS service needs to remain for things to keep working.

```suggestion
            'interpretaties:quartaire_stratigrafie', QuartairStratigrafie)
```

File: pydov/search/interpretaties.py [hunk 3]
```diff
@@ -745,3 +746,126 @@ def search(self, location=None, query=None, return_fields=None):
             columns=GeotechnischeCodering.get_field_names(
                 return_fields))
         return df
+
+
+class QuartaireStratigrafieSearch(AbstractSearch):
```
Comment (Roel) [inline pydov/search/interpretaties.py]:
```suggestion
class QuartairStratigrafieSearch(AbstractSearch):
```

File: pydov/types/interpretaties.py [hunk 4]
```diff
@@ -722,3 +722,97 @@ class GeotechnischeCodering(AbstractBoringInterpretatie):
         'sourcefield': 'Y_mL72',
         'type': 'float'
     }]
+
+
+class QuartaireStratigrafieLaag(AbstractDovSubType):
```
Comment (Roel) [inline pydov/types/interpretaties.py]:
```suggestion
class QuartairStratigrafieLaag(AbstractDovSubType):
```

File: pydov/types/interpretaties.py [hunk 5]
```diff
@@ -722,3 +722,97 @@ class GeotechnischeCodering(AbstractBoringInterpretatie):
...
+
+    _name = 'quartaire_stratigrafie_laag'
```
Comment (Roel) [inline pydov/types/interpretaties.py]:
```suggestion
    _name = 'quartairstratigrafie_laag'
```

File: pydov/types/interpretaties.py [hunk 6]
```diff
@@ -722,3 +722,97 @@ class GeotechnischeCodering(AbstractBoringInterpretatie):
...
+        'sourcefield': '/lid2',
+        'definition': 'tweede eenheid van de laag quartairstratigrafie. '
+                      'Indien niet ingevuld wordt default dezelfde waarde '
+                      'als voor Lid1 ingevuld',
+        'type': 'string',
+        'notnull': False
+    }]
+
+
+class QuartaireStratigrafie(AbstractCommonInterpretatie):
```
Comment (Roel) [inline pydov/types/interpretaties.py]: Quartairstratigrafie interpretaties are always linked to Boringen, so should be a subclass of AbstractBoringInterpretatie.

```suggestion
class QuartairStratigrafie(AbstractBoringInterpretatie):
```

File: pydov/types/interpretaties.py [hunk 7]
```diff
@@ -722,3 +722,97 @@ class GeotechnischeCodering(AbstractBoringInterpretatie):
...
+    In Dutch: Afgeleid type van interpretatie, specifiek voor de quartaire
+    stratigrafie. Een Quartair interpretatie omvat een discrete formele
+    interpretatie van het materiaal in de ondergrond, al dan niet aan
+    de hand van monsters, op basis van een door DOV gestandaardiseerde
+    codering. De Quartair interpretatie gebeurt in tegenstelling tot de
+    formele interpretatie op basis van sedimentgenetische profieltypen
+    ipv lithostratigrafie
+    """
+
+    _subtypes = [QuartaireStratigrafieLaag]
```
Comment (Roel) [inline pydov/types/interpretaties.py]:
```suggestion
    _subtypes = [QuartairStratigrafieLaag]
```

**Comment by pjhaest** [comment]
in progress - not complete yet

**Pull request merged**